cpp/modmesh/pilot/RManager.cpp

```diff
@@ -86,7 +86,7 @@ R3DWidget * RManager::add3DWidget()
     R3DWidget * viewer = nullptr;
     if (m_mdiArea)
     {
-        viewer = new R3DWidget();
+        viewer = new R3DWidget(/*window*/ nullptr, /*scene*/ nullptr, /*parent*/ m_mdiArea);
```

Comment:
Add parent.  Hope this makes lifecycle correct.  It can potentially leak resources and should be reviewed in the future.